nodebuilder/node/module.go

```diff
@@ -1,14 +1,16 @@
 package node
 
 import (
+	"github.com/cristalhq/jwt"
 	"go.uber.org/fx"
 )
 
 func ConstructModule(tp Type) fx.Option {
 	return fx.Module(
 		"node",
-		fx.Provide(func() Module {
+		fx.Provide(func(secret jwt.Signer) Module {
```

Comment:
This will be actually used in a later PR (where node module will be able to generate new tokens and verify perms of tokens w the given secret